geo-types/Cargo.toml

```diff
@@ -21,7 +21,7 @@ use-rstar_0_9 = ["rstar_0_9", "approx"]
 
 [dependencies]
 approx = { version = ">= 0.4.0, < 0.6.0", optional = true }
-arbitrary = { version = "=1.1.3", optional = true }
+arbitrary = { version = "1.2", optional = true }
```

Comment:
I was convinced to [prefer precise version constraints](https://users.rust-lang.org/t/psa-please-specify-precise-dependency-versions-in-cargo-toml/71277).